packages/injected/src/types.ts

```diff
@@ -110,7 +113,8 @@ export interface CustomWindow extends Window {
   xfi: {
     ethereum: InjectedProvider
   }
-  gamestop: InjectedProvider
+  gamestop: InjectedProvider,
+  bitkeep: InjectedProvider
```

Comment:
Since you are referencing `window.bitkeep.ethereum` in your provider on ln 522 of wallet.ts (`provider: window.bitkeep && window.bitkeep.ethereum,`) you may need to follow the xfi pattern above (line 113 in packages/injected/src/types.ts)
```suggestion
  bitkeep: {
    ethereum: InjectedProvider
  }
```

Reply:
Just a heads up,  I fixed these on your fork and pushed!